src/app/callstats/callstats.config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "AppID": "724446896",
+  "AppSecret": "/BKw/++t+DXe:UCxvwuXdkE6AIuxHXRj5PnaKLNeAgNZdp6rGShdWNmc="
+}
```

Comment:
This should not be part of the PR, of course.

Reply:
@ancorgs Done!